### DISCUSSION TITLE
Bisakah mengubah url halaman menjadi page ID?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#244] [Q&A] [answered]
kang, ada cara untuk mengubah url halaman menjadi menjadi url yang menggunakan page.id ?

**Comment by taufik-nurrohman** [comment, marked as answer]
Bisa, tapi kemungkinan akan mengalami konflik dengan page offset. Dan untuk mencari ID laman itu lebih membutuhkan banyak tenaga dibandingkan dengan mencari lokasi laman dari jalur berkas. Kecuali kalau data laman disimpan di dalam basis data.

~~~ php
$id = $_GET['id'] ?? "";
foreach (g(LOT . D . 'page', 'page', true) as $k => $v) {
    $page = new Page($k);
    if ($id === $page->id) {
        test($page);
        exit;
    }
}
~~~
